src/execution.jl

```diff
@@ -56,6 +73,14 @@ This is a low-level call, prefer to use [`cudacall`](@ref) instead.
     _launch(f, griddim, blockdim, shmem, stream, args)
 end
 
+@inline function launch{N}(f::CuFunction, griddim::CuDim3, blockdim::CuDim3,
```

Comment:
Just make this a 1-line definition calling `launch`, don't duplicate its body to call `_launch`.
Also add the second calling syntax to the documentation above, and mention the default values for those arguments.